test/functional/core/job_spec.lua

```diff
@@ -175,63 +168,71 @@ describe('jobs', function()
   end)
 
   it("will not buffer data if it doesn't end in newlines", function()
-    if helpers.pending_win32(pending) then return end  -- TODO: Need `cat`.
     if os.getenv("TRAVIS") and os.getenv("CC") == "gcc-4.9"
       and helpers.os_name() == "osx" then
       -- XXX: Hangs Travis macOS since e9061117a5b8f195c3f26a5cb94e18ddd7752d86.
       pending("[Hangs on Travis macOS. #5002]", function() end)
       return
     end
 
-    nvim('command', "let j = jobstart(['cat', '-'], g:job_opts)")
+    nvim('command', "let j = jobstart(has('win32') ? ['find', '/v', ''] : ['cat', '-'], g:job_opts)")
     nvim('command', 'call jobsend(j, "abc\\nxyz")')
-    eq({'notification', 'stdout', {0, {'abc', 'xyz'}}}, next_msg())
-    nvim('command', "call jobstop(j)")
-    eq({'notification', 'stdout', {0, {''}}}, next_msg())
+    eq({'notification', 'stdout', {0, iswin() and {'abc', ''} or {'abc', 'xyz'}}}, next_msg())
+    nvim('command', "call jobclose(j)")
     eq({'notification', 'exit', {0, 0}}, next_msg())
   end)
 
   it('preserves newlines', function()
-    if helpers.pending_win32(pending) then return end  -- TODO: Need `cat`.
-    nvim('command', "let j = jobstart(['cat', '-'], g:job_opts)")
-    nvim('command', 'call jobsend(j, "a\\n\\nc\\n\\n\\n\\nb\\n\\n")')
-    eq({'notification', 'stdout',
-      {0, {'a', '', 'c', '', '', '', 'b', '', ''}}}, next_msg())
+    nvim('command', "let j = jobstart(has('win32') ? ['find', '/v', ''] : ['cat', '-'], g:job_opts)")
+    -- find.exe can't handle 3+ consecutive newlines consistently
+    nvim('command', 'call jobsend(j, "'..(iswin()
+      and [[a\n\nc\n\n]]
+      or  [[a\n\nc\n\n\n\nb\n\n]]
+    )..'")')
```

Comment:
Would rather keep the test pending than have a lot of special-case code that is difficult to follow, just to get the test working.  Maybe something can be improved in the future so that the test can pass without requiring lots of weird workarounds.